### DISCUSSION TITLE
6.x branch seems to be missing (a) commit(s)?

### DISCUSSION THREAD

**Discussion by nvaken** [deployphp/deployer#2389] [General]
When checking out the branch `6.x` I'm unable to find some code which I've previously committed (30552d) to this repository. As far as I can tell, this is 6.x specific code (`src/Ssh/Arguments.php` is not even present in 7.x (`master`) anymore). Am I going bonkers? Or is this actually missing? I would like to [introduce a fix](https://github.com/deployphp/deployer/issues/2388) for that specific few lines of code.

**Comment by antonmedv** [comment]
Code what refactored and move to https://github.com/deployphp/deployer/blob/master/src/Component/Ssh/Client.php#L231-L264

**Reply by nvaken** [reply]
Thanks @antonmedv, but I'm really searching for a possibility to update the `6.x` branch, since all of our projects currently depend on the "old way" of importing hosts config (`inventory()`). Of course, I'd be glad to _also_ apply the fix to the file you mentioned `Component/Ssh/Client.php`.
